client/CropEditPanel.js

```diff
@@ -152,6 +154,16 @@ class CropEditPanel extends React.Component {
             'Water',
             practicalplants.PP_WATER_VALUES,
             initialWater
+          ),
+          this.getSingleDropdownElement(
+            'Family',
+            practicalplants.PP_FAMILY_VALUES,
+            initialFamily
+          ),
+          this.getSingleDropdownElement(
+            'Genus',
+            practicalplants.PP_GENUS_VALUES,
+            initialGenus
```

Comment:
These are name properties so it'd be easier to follow if they were displayed right after common name and binomial name.

Reply:
Sure